admin/section/class-convertkit-settings-base.php

```diff
@@ -461,7 +461,7 @@ public function get_description( $description ) {
 		}
 
 		// Return description lines in a paragraph, using breaklines for each description entry in the array.
-		return '<p class="description">' . implode( '<br />', $description );
+		return '<p class="description">' . implode( '<br />', $description ) . '</p>';
```

Comment:
@noelherrick FYI looks like I never closed the paragraph tag for other settings fields, so fixing that here too.

Reply:
Nice!